.changeset/soft-rabbits-draw.md

```diff
@@ -0,0 +1,22 @@
+---
+"astro": patch
+---
+
+**BREAKING CHANGE to the experimental Container API only**
+
+Until now, the Container API was rendering the components as they were full-fledged Astro pages.
+
+Full-fledged Astro pages contain `<!DOCTYPE html>` but default. This was not intended, and this patch fixes the behavior. 
```

Comment:
```suggestion
Changes the default page rendering behavior of Astro components in containers, and adds a new option `partial: false` to render full Astro pages as before.

Previously, the Container API was rendering all Astro components as if they were full Astro pages containing `<!DOCTYPE html>` by default. This was not intended, and now by default, all components will render as [page partials](https://docs.astro.build/en/basics/astro-pages/#page-partials): only the contents of the components without a page shell.
```
I'm assuming these will work like "page partials" that we've already documented, so we can link there if that's helpful!